CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ All notable changes to this project will be documented in this file.
  * `TileManager` trait and demonstration of locally generated tiles.
  * Zoom and drag gestures can now be disabled.
  * Add `gesture_handled` to `Plugin::draw()` to let plugins know if the gesture was handled by the map.
+ * `egui` updated to 0.25.
```

Comment:
Could you move this to "Unreleased"? I can then make another release, with egui update only.

Reply:
done